packages/eslint-plugin/package.json

```diff
@@ -80,7 +80,7 @@
     "typescript": "*"
   },
   "peerDependencies": {
-    "@typescript-eslint/parser": "^5.0.0",
+    "@typescript-eslint/parser": "^5.0.0 || ^6.0.0 || ^6.0.0-alpha",
```

Comment:
@bradzacher you're right, `semver.satisfies("6.0.0-alpha81", "^6.0.0")` is `false`...! I don't know a better way than this?

Reply:
I'd suggest `>=5.0.0 || <7.0.0-0`.

Reply:
That or don't even set a top end; what's the worst that could happen? (actually though)

Reply:
Once in a while someone tries out the projects with a weirdly out of sync set of versions - like a 2.x and a 5.x. I like the `<7`!

Reply:
I'm wrong; prerelases simply do not match _any_ version. So ^6 will be equivalent.

```
> semver.satisfies("6.0.0-alpha81", "<7")
false
```

(This is why I use overrides in typescript and other packages when testing TS 5.0 or other prereleases)

Reply:
I was thinking of TS's own `Version` type, which has slightly different behavior because it _does_ actually need to order prereleases for things like typesVersions and whatnot.

Reply:
Basically: this sucks

Reply:
🫠